Lightbox.js

```diff
@@ -34,11 +34,13 @@ var Lightbox = React.createClass({
       friction:      PropTypes.number,
     }),
     swipeToDismiss:  PropTypes.bool,
+    transform:       PropTypes.bool,
```

Comment:
Can we rename this to something more descriptive to the end user? Maybe something like: `pinchToZoom`.